core/templates/services/UpgradedServices.ts

```diff
@@ -475,6 +475,8 @@ import { SubtopicPageObjectFactory } from
   'domain/topic/SubtopicPageObjectFactory';
 import { SubtopicViewerBackendApiService } from
   'domain/subtopic_viewer/subtopic-viewer-backend-api.service';
+import { SuggestionModalForCreatorDashboardBackendApiService } from
+  'pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-dashboard-backend-api.service';
```

Comment:
We don't add services here anymore. Can you undo the changes in this file?

Reply:
Done.